dags/operational_monitoring_backfill.py

```diff
@@ -0,0 +1,80 @@
+import datetime
+
+from airflow.decorators import dag, task
+from airflow.models.param import Param
+
+from operators.gcp_container_operator import GKEPodOperator
+from utils.tags import Tag
+
+docs = """
+### operational_monitoring_backfill
+
```

Comment:
```suggestion
Build from telemetry-airflow repo, [dags/operational_monitoring_backfill.py](https://github.com/mozilla/telemetry-airflow/blob/main/dags/operational_monitoring_backfill.py)
```

In bigquery-etl this step is automated but putting it in manually is quicker for now :)